server/users.ts

```diff
@@ -1339,8 +1340,7 @@ export class User extends Chat.MessageContext {
 		}
 		// cancel tour challenges
 		// no need for a popup because users can't change their name while in a tournament anyway
-		for (const roomid of this.games) {
-			const room = Rooms.get(roomid);
+		for (const room of Rooms.rooms.values()) {
 			// @ts-ignore Tournaments aren't TS'd yet
 			if (room.game && room.game.cancelChallenge) room.game.cancelChallenge(this);
 		}
```

Comment:
```suggestion
		for (const game of this.getGames()) {
			(game as Tournament)?.cancelChallenge(this);
		}
```

Reply:
If `Tournament` isn't a global type we may need to import it from somewhere. `as any` might be easier.